bt_ddos_shield/validators_manager.py

```diff
@@ -0,0 +1,26 @@
+from abc import ABC, abstractmethod
+
+class ValidatorsManager(ABC):
+    """
+    Abstract base class for manager of validators and their public keys used for encryption.
+    """
+
+    def __init__(self):
+        pass
+
```

Comment:
I really can't feel that class, but I can't put my finger on it. Lets see where it goes.

Reply:
At least there is refresh_validators method, which could collect new validators list in some future (not hardcoded dict) implementation. So i made it like that - not with dict interface, about which we were talking.